README.md

```diff
@@ -15,18 +15,13 @@ jobs:
       - uses: pmorelli92/github-container-registry-build-push@1.0.1
         name: Build and Publish latest service image
         with:
-          # Read note below to see how to generate the PAT
-          personal-access-token: ${{secrets.GHCR_PAT}}
+          personal-access-token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
I agree with this change. I would however go and rename the variable from `personal-access-token` to `github-push-secret` or something like that. Same in the `entrypoint.sh`. This will indeed cause me to release a major version but I guess we can live with it :)